vmdb/app/models/ems_microsoft.rb

```diff
@@ -1,6 +1,7 @@
-
 $:.push(File.expand_path(File.join(Rails.root, %w{.. lib Scvmm})))
 
+require 'gssapi'
```

Comment:
@djberg96 while this is correct, I'm concerned that gssapi is specific to the methods that use winrm to connect to the remote system.  It's not clear which methods have a dependency on gssapi.  If for example, we wanted to extract the WinRM interface out at some point, it's not clear how to do this.  Do you have any thoughts?